tutorials/applications/Using QAOA to solve a MIS problem.ipynb

```diff
@@ -15,6 +15,7 @@
    "source": [
```

Comment:
<p>good idea to write the formula, but sorry I'm quite allergic to arrows over vectors, maybe remove them ? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pasqal-io/Pulser/pull/93/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>